devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/directive-forest/directive-forest.component.html

```diff
@@ -37,15 +37,25 @@
             </mat-icon>
           </button>
         }
-        <span class="element-name" [class.angular-element]="isElement(node)">{{ node.name }}</span>
+        <span
+          class="element-name"
+          [class.non-element]="!node.original.resolutionPath"
+          [class.angular-element]="node.original.component?.isElement"
+        >
+          {{ node.name }}
+        </span>
```

Comment:
```suggestion
        >{{ node.name }}</span
      >
```

The updated formatting adds spaces to the element name which break the tag-like visualization in the app.